ov/id_pools_ipv4_range.go

```diff
@@ -193,116 +186,125 @@ func (c *OVClient) GetFreeFragments(filter string, sort string, start string, co
 	return freeFragments, nil
 }
 
-func (c *OVClient) CreateIPv4Range(ipv4 CreateIpv4Range) error {
+func (c *OVClient) CreateIPv4Range(ipv4 CreateIpv4Range) (Ipv4Range, error) {
 	log.Infof("Initializing creation of ipv4Range for %s.", ipv4.Name)
 	var (
-		uri = "/rest/id-pools/ipv4/ranges/"
-		t   = (&Task{}).NewProfileTask(c)
+		uri     = "/rest/id-pools/ipv4/ranges/"
+		iprange Ipv4Range
 	)
 	// refresh login
 	c.RefreshLogin()
 	c.SetAuthHeaderOptions(c.GetAuthHeaderMap())
 
-	t.ResetTask()
 	log.Debugf("REST : %s \n %+v\n", uri, ipv4)
-	log.Debugf("task -> %+v", t)
 	data, err := c.RestAPICall(rest.POST, uri, ipv4)
 	if err != nil {
-		t.TaskIsDone = true
 		log.Errorf("Error submitting new ipv4Range creation request: %s", err)
-		return err
+		return iprange, err
 	}
 
 	log.Debugf("Response New ipv4Range %s", data)
-	if err := json.Unmarshal(data, &t); err != nil {
-		t.TaskIsDone = true
+	if err := json.Unmarshal(data, &iprange); err != nil {
 		log.Errorf("Error with task un-marshal: %s", err)
-		return err
+		return iprange, err
```

Comment:
I think POST request does not provide response body. Hope you have checked and confirmed that you are getting payload post creation.

Reply:
in this case it returns the response body.